index.html

```diff
@@ -71,7 +71,7 @@ <h2>Add New Recipe</h2>
       <button type="button" id="add_more_ingredient">Add More Ingredient</button
       ><br /><br />
 
-      <button type="submit">Add Recipe</button>
+      <button type="add recipe">Add Recipe</button>
```

Comment:
```suggestion
      <button type="submit">Add Recipe</button>
```
"add recipe" is not a valid `type`. You can see the list of allowed values here: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/button#type